renovate.json

```diff
@@ -53,6 +53,6 @@
       "automerge": true
     }
   ],
-  "ignoreDeps": ["styled-components"],
+  "ignoreDeps": [],
```

Comment:
I think we should add Emotion here

Reply:
It shouldn't be affected because of the `ignorePaths` defined under this, but I'll add it just to be safe!

Reply:
Ahh good point. I see it now